xcell/mappers/mapper_ACT_base.py

```diff
@@ -18,6 +22,18 @@ def _get_ACT_defaults(self, config):
         self.nl_coupled = None
 
     def get_signal_map(self):
+        """
+        Returns the signal map of the children \
```

Comment:
It returns the signal map of the instance of the class you have. Remove the references to parent and child classes